.gitignore

```diff
@@ -65,5 +65,9 @@ public
 yarn-error.log
 .pnp/
 .pnp.js
+
 # Yarn Integrity file
 .yarn-integrity
+
+# Yarn Workspace Directories
+western-up-react
```

Comment:
Is this needed? I didn't think we were using workspaces here anymore.

Reply:
I've been using workspaces for the initial development of the components, that's why this is here